website/content/docs/connect/intentions.mdx

```diff
@@ -97,6 +97,9 @@ accepted.
 An intention source or destination may also be the special wildcard
```

Comment:
```suggestion
You can use the `*` wildcard when defining an intention source or destination. The wildcard matches _any_ value and can serve as a "catch-all" entry for intentions that should have a wide scope.     
```

This suggestion about using language more consistent with our style guide and to define the purpose of a "catch-all" entry. Please let me know if it is inaccurate and I can try providing a better suggestion.